server/src/index.js

```diff
@@ -0,0 +1,52 @@
+require('dotenv').config();
+const express = require('express');
+const app = express();
+const morgan = require('morgan');
+const bodyParser = require('body-parser');
+const path = require('path');
+const cors = require('cors');
+const mongoose = require('mongoose');
+const expressJwt = require('express-jwt');
+
+const server = require('http').Server(app);
+
+// MongoDB connection configuration
+mongoose.Promise = global.Promise;
```

Comment:
looks as a mongoose example copy. This is necessasry and can be applied directly `mongoose.Promise = Promise` and one more, I'm not sure that this action is still necessary:) can works by default